hazelcast/src/test/java/com/hazelcast/test/TestHazelcastInstanceFactory.java

```diff
@@ -187,7 +186,7 @@ protected static Address createAddress(String host, int port) {
 
     private static Config init(Config config) {
         if (config == null) {
-            config = new XmlConfigBuilder().build();
+            config = new Config(); //new XmlConfigBuilder().build();
```

Comment:
great catch, thanks! 
I have no idea, it must have slipped through when I was rebasing the original changes!